src/Application/Writer/WriteVisitor.php

```diff
@@ -191,4 +193,9 @@ public function leaveNode(Node $node)
 
         return $node;
     }
+
+    public function getAdditionalUsages(): array
```

Comment:
this should be removed as this should be handled by php-cs-fixer/rector to convert FCQN to use statements.

Reply:
that does not work with the writer of the library

Reply:
fixed this via rector. the writer need to use the `\` prefixed class else it tries to use a relative class to the current namespace.